docs/release-notes/2.22.0.md

```diff
@@ -0,0 +1,44 @@
+# Percona Monitoring and Management 2.22.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">September 21, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## New Features
+
+- [PMM-8396](https://jira.percona.com/browse/PMM-8396): Backup Management: Retry failed scheduled backups
+- [PMM-8079](https://jira.percona.com/browse/PMM-8079): DBaaS: Upgrade Operators inside Kubernetes cluster
+- [PMM-7924](https://jira.percona.com/browse/PMM-7924): MySQL Performance Details dashboard: Add "Performance Schema Status Monitoring" chart
+
+## Improvements
+
+- [PMM-8688](https://jira.percona.com/browse/PMM-8688): Backup Management: Action button "Create Backup" mislabeled
+- [PMM-8311](https://jira.percona.com/browse/PMM-8311): Integrated Alerting: Disable edit/delete buttons for Percona-sourced Templates
+- [PMM-8077](https://jira.percona.com/browse/PMM-8077): DBaaS: Show the current version of Operators used in Kubernetes
+- [PMM-8509](https://jira.percona.com/browse/PMM-8509): "Check for updates" can be disabled in Configuration->Settings->Advanced settings
```

Comment:
Aren't we added a flag to force Enable this too over Env variables? 


This is for me more like: "Let Disable Update follow the standard parameter process for PMM variables. be able to specify parameter over API. env var and UI"

?.

Reply:
This suggestion is very confusing. It sounds like the option to Disable Update is now present in the web GUI as well as a config file, correct? It looks like this fix is specific to the web GUI, so the parameters flags may not make much sense to the user, if that is not a new functionality.

Reply:
@JiriCtvrtka ^

Reply:
Fix working for UI, API params and env var. Same behavior like in case of telemetry.

@denisok I believe we missing https://jira.percona.com/browse/PMM-8421 in release notes

Reply:
```suggestion
- [PMM-8509](https://jira.percona.com/browse/PMM-8509): Management of ability to update PMM Server in the same way as this implemented to other Settings for PMM. Users can use API, UI, or docker Environment Variables to change the setting responsible for the Update process.  As with all PMM settings, the Environment Variable way have higher priority and can't be changed over API or UI 
```

Reply:
yes, PMM-8421 needs to be resolved first and script should pick it up in the next round. @PaulJacobs-percona fyi.

Reply:
Since Roma has manually reordered items, I won't be using Jirabot to re-extract RNs. Instead I will copy/paste the missing item in the section where Jirabot would list it and let Roma shuffle it up or down.

Reply:
ok, that one is fixed, please add it

Reply:
https://github.com/percona/pmm-doc/pull/588/files#r708133689
Suggested change above is IMO too long. Details should be moved to the ticket description.

Reply:
In this case, this is an important change in the Set-Up process, so added here to be more visible.